Client/Assets/Search/SearchPlugins/google-b-1-m.xml

```diff
@@ -5,7 +5,7 @@
="16" height="16">data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGAAAABgCAYAAADimHc4AAAACXBIWXMAACxLAAAsSwGlPZapAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAg6SURBVHgB7ZxfbFtXHce/5147dv47I3/aMZg7EEPin4EkrDDo7Z8HnqhLh1AZUmIxJNQ+tPDCJDTF0WC8gNJIiD3wEFcaagvbmkigPSxNbiUmrU2giYSQ0KbF0/40TarZid0ls33v2TluPXVtMvv6nvsnyfm8XOfa19f5fc/5nd/vd865gEQikUgkEolEIpFIJBKJRCKRSCSS7Q7BFuBtrS/WoFANhEQpyIMENHb7rWjlM+xclr2XZS/T5SPFvGIqeh7huT26noVP8aUAC5oWaVbyg8zgh7mxmUEjsAPBHBPkUsFA6gF9Zg4+wlcCXNMe0QKqeRIwNdtG3wwuhonR7qmZFHyALwTghldVY4i91OAeadYrhr0WwlMBmOGjzPBjcNfwd+OpEJ4JsHSw7yTz70nHXI1FFCBVNNTh3fqrabiI6wL4pNVvRtow1P1uiqDARZZYOMmMPw1/Gp/DG8fC4oFvnYJLuCbA6s+ODRCVcuNH4XMUYo4ss8AALuCKC1o62D8A0BS2CJSS4Z6pK0m4gOMCXD/YH2eD7QVsEdw0PsdRAfiAG1BLV/0S6VTDbeNzAnCI29HOtBPGJy2toPlcmvAYvkJbJILVLKsVoa77eWF8jmMCBNXSiAkShTh0k9IJxSR618TUpvWczNhYtPDcn1ixTokTSvaB0Gi1L/bK+BxHXBBPstjhNGzCflyWdPecUR/+aqoj+UxdRbSlA32D7JuGNhPCS+NzhAsg0O/rLClKiEqKrh/qTbIeMXTnOa+NzxGeBzC/b6u8wFu9Segvuy/OCM1IeyZnk0zQPaAkzf/2g/E5QnvA7YF3AfXCjBP83qEj9bqbWuC/kQRK8V2Ts7ZdpAiEClB4OZhav/jpgfWZLliGGd8wlf1uF8O8RoUg1qZZHYWqY8GHclDCBopvtNV8rdLSli2tl/buNONzhI0BaimYrLwO9S2j/fj/oLQXarrWyOcSO9H4HGECKAT7PvY3M37r46+j4Qsr1S4d7bl4ZRw7FCEClC73x+kGVU4uQvPRBTR+d3HjC5nfb/3pcV8Mhl4hRABz9Wr8k94PP7pY7g13uyQDdLgxkUhjByMkCipNBhdoDXV+c6UBub9+vnzkrb976soe7HBs9wAe/dRi/PLNWA/ggzN3SeqnOnVI7BfjgoYao9YuKbsk0rU+ir/DNoeezmmmogzAA0zQUf03LbaSRtsCsJQ+ZtWR8TJyy4llIdmuqfAiGx2EBygU8+xg6/+wPwg37IpZvcSgZB7bAEKrl7qrYV+A4vV2WIRNUerYBlBFsfy/3419AYKdliufCjHT2AZQP/QAUrphWQA2bvh2ubgVCCFR2MTVhVmSe5ECeIx9AdT7YBUR0cN2wb4AipKGRUyibIl1QtVgg3AaNrEtgFm48SYswkrXX4OkjP0oKPRgGtbRsA24r5lUneyohm0BqNKYhkV48Y6+FI5ii5PJm7bLKbZrQUbhNd3ql5y++WWcf3/PIPBiEjYxi61sNm1dh03UoHGBNQxLZRVTpbYFEDIfULzUlUExW3VgvWY24tcr/XjNaOd3zs7++IUO+IQDv8tnYHFd6TfuV/f/IdGowwZC8gBCwhPVPvOfYidOZL9zy/gcikjf2R8OwgdorKSNOhb12jU+R0wiFtr9iZPq59c/hxMr32Y9oOnjbxAyFBuLex6SspLCICxCKXQIQIgAav9lnR3uqe/kaBBPrvbjdP5LG17HB+NAWHFtP9ZGaL9fixICyxM6zHdX7fW1IMgFkSwrMZ+58xz39wOZfbhU2FXt4qHes49p8AhiGEnUgUl81AM4tF37yA3984PPMuNr97qcTSCEjj1yNh6Fy7CBd7Ce1k9B0nanIisIEyDYN6mzg85DzN/mYmX3UyvcFRlEnXZTBO562GEEdUBBhyEIodXQvcuHh8+vPYR6cFMEbnzFNPmW2foCgKKqQxBCBZg99rzODjrqpCKCk+EpDzkV07jK7hZFXZCUnrSe/W+G8PmAADUSPMlCnZTLFISMffPcUaHjgsbC3f7nnhgx2l+pv+WDZ96KMPfDcWSPWO+5x1hoSevyr3fCl69QQscDpjn66rHxNOqAGz7fpJxk33aKJ3/8XOi9OEKZI7CMSYennmpNQiCO7RPuPXdU9DMhxtlEzoQKU68mRt/f4jFaCmhQ6OHNfoPKIrXGxZNQSp2oBR750GLT1/Wk2PlsxwTg7qMp1HZ1tXBTfKbLXRy9tSCqsle47LqYX2c5SaTS0qvBjd+49HOoa1+s+tmHe8iRZ59oFr6M3tGd8r94Oan9f/WN6VzxJvxMVZfkgOup4PizInhEwwdV+Jxg7lGEbzwOclfy2N6I+Qu/arG8+q9WHF8VMXPsxVRv11eERg5OUGz9F25+5imYgRsfneN+/wd7m+NwENeemBX/x4lTb+cXbUdGTsN7QCgTRzD7fTboKvtFxvwb3g8u8szlP8cvLEyOsJYVhY9h0db88egL8cTeW5u6Hb0XXIZHRwZRpv0qQmeo48xL8b+c4hVeuIBnT01kJegka2pD8As8tDXI8OxPnnd106Cnzw3lvWF32wPTb+WuReEhHaE2PbeeSdSbbdvBF0/OLRffCIY8cEs6KGv1t4qInuCrZ0c/+cofB/+99N+BzAerGhyiNdiMgKLqmfWcp4av4Munp5cHaigaS+D4bJUGuzD/3tHQNtcabJp4b/mdlJ4Y983+BF8KcCeU0kjfuR/F7m/q1N5dW+ZrSiNtDc2xjWpM7Hx2pZBn89NkrjMcyTYFwvPdoY65Zw89PedWVCORSCQSiUQikUgkEolEIpFIJBKJD/kQkCH65NcnDzgAAAAASUVORK5CYII=</Image>
```

Comment:
Shouldn't search engine icon changes be in another PR?

Reply:
this doesn't change how the Search Engines work, like having theme-able images, it just replaces the Google logo image with one without a background so it conforms with the design

Reply:
Oh ok, that image was provided by design?

Reply:
indirectly, it was stolen from https://www.figma.com/file/K9m9aQx40RmWnXnIc1GDeQ/iOS-%26-iPad-UI?node-id=1162-108085&t=sT3eJyyZgh4PLTip-0